.ci/enable_sandbox_cgroup_only.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+handle_error() {
```

Comment:
This function is duplicated in other scripts - could it be put into `.ci/lib.sh` or similar?